deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -420,9 +420,14 @@ cephBlockPools:
 
         # RBD image format. Defaults to "2".
         imageFormat: "2"
-        # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
+
+        # RBD image features, equivalent to OR'd bitfield value: 63
+        # Available for imageFormat: "2". Older releases of CSI RBD
+        # support only the `layering` feature.  The Linux kernel (KRBD) supports the
+        # full complement of features as of 5.4
```

Comment:
Any reason not to make this consistent with the example storageclass.yaml?

Reply:
Done.  Missed that when I reworded.  I obsess a bit over wording.